### PR TITLE
Adds global constants

### PR DESCRIPTION
**Pull request by damirka**
Closes https://github.com/AleoHQ/leo/issues/1147

### CODE REVIEW

File: documentation/developer/language/02_variables.md [hunk 1]
```diff
@@ -35,3 +36,33 @@ const a: u32 = 0;
...
+    // constants are accessible everywhere
+    let num = MAGIC_NUMBER;
+}
+```
+
+:::info
+Global constants only exist in the scope of the file where they are defined. Import of these constants is not allowed.
+:::
+
+Constants can also be used in the other constant definitions:
```
Comment (gluax) [inline documentation/developer/language/02_variables.md:63]: Can you specify here that using constants in other constants is top-down only? I.e., can define a const that uses a const after that one is defined.

**Review by gluax — CHANGES_REQUESTED** [review]

File: documentation/developer/language/02_variables.md (hunk 1 above)
Comment (acoglio) [reply documentation/developer/language/02_variables.md:63]: Yes, important point.

File: documentation/developer/language/02_variables.md [hunk 2]
```diff
@@ -35,3 +36,33 @@ const a: u32 = 0;
...
+## Global Constants
+
+Leo allows defining global constants in the main scope of a program. Syntax for these definitions is the same as for
+constants inside the function:
+
+```leo
+const MAGIC_NUMBER: u8 = 42;
+
+function main() -> u8 {
+    // here you can access these variables
```
Comment (acoglio) [inline documentation/developer/language/02_variables.md:49]: I would change ‘variables’ to ‘constants’ in this comment.

**Review by acoglio — APPROVED** [review]
Modulo Jon’s and my suggestions, this looks good.

**Comment by damirka** [comment]
@acoglio pushed fixes. Should be better now.

**Review by acoglio — APPROVED** [review]
Looks good.

**Review by gluax — APPROVED** [review]
LGTM

**Pull request merged**